.github/workflows/pr.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Semantic PR"
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            fix
+            feat
+            chore
+          validateSingleCommit: true #single commit can ovveride squash merge commit message
```

Comment:
think you can remove this now, but make sure repo is following instructions in handbook